app/default/data.py

```diff
@@ -22,6 +24,11 @@
     current_datetime_before_given_iso_string,
 )
 
+
```

Comment:
So you're looking to get a ttl hash for...5 minutes?

Reply:
yes that's right